functional_tests/scylla_operator/test_functional.py

```diff
@@ -790,3 +790,40 @@ def test_operator_managed_tls(db_cluster: ScyllaPodCluster, tmp_path: path.Path)
         output = res.all()
         assert len(output) == 1
         log.debug(output)
+
+
+@pytest.mark.required_operator("v1.8.0")
+@pytest.mark.requires_tls
+def test_cloud_bundle_connectivity_python(db_cluster: ScyllaPodCluster):
+
+    assert db_cluster.connection_bundle_file, "cloud bundle didn't found"
+
+    with db_cluster.cql_connection_patient(db_cluster.nodes[0]) as session:
+        for host in session.cluster.metadata.all_hosts():
+            log.debug(host)
+        res = session.execute("SELECT * FROM system.local")
```

Comment:
Looks like lines 804-807 have wrong indentation. Looks like it must be part of the above loop.

Reply:
no it's not part of the loop, the loop is just for printing debug information
the loop can be remove

Reply:
removed